contrib/cirrus/runner.sh

```diff
@@ -55,31 +60,66 @@ function _run_unit() {
 }
 
 function _run_apiv2() {
+    if _only_changes_are_in docs hack test/{buildah-bud,e2e,system,upgrade}
```

Comment:
A much smarter person than I would come up with a way to use positive logic instead of negative, e.g. "if this is a doc- or hack- or test-only change, except for changes within test/apiv2 itself". Suggestions welcome.

Reply:
This is probably best left up to a human to decide, e.g. by adding `[CI:DOCS]` magic or using similar mechanism.  In extreme cases, the tasks can be manually-triggered.